locale/admin-docs.pot

```diff
@@ -12017,22 +12017,22 @@ msgid "Enter the \"APP ID\" and the \"APP SECRET\" from the Gitlab OAUTH Applica
 msgstr ""
 
 #: ../settings/security/third-party/gitlab.rst:27
-msgid "Please ensure to use ``https://{git_host}/api/v4/`` for site."
+msgid "Please ensure to use ``https://{git_host}/`` without any ``/api/v4`` suffix for site."
```

Comment:
I would not say something about the suffix, because for "new" installations it could be confusing, this sentence would only be relevant for older instances.